src/flam3util.h

```diff
@@ -24,7 +24,7 @@
 
 #undef VERSION
 extern "C" {
-#include "flam3.h"
+#include "flam3/flam3.h"
```

Comment:
The INCLUDEPATH in qosmic.pro or your pkg-config can adjusted to locate flam3.h on your system.
I think this change should be removed from your request.